core/monitoring/user-events/src/main/scala/org/apache/openwhisk/core/monitoring/metrics/PrometheusRecorder.scala

```diff
@@ -208,12 +209,7 @@ object PrometheusRecorder extends PrometheusMetricNames {
       initiatorNamespace,
       actionName)
   private val memoryGauge =
-    histogram(
-      memoryMetric,
-      "Memory consumption of the action containers",
-      actionNamespace,
-      initiatorNamespace,
-      actionName)
+    gauge(memoryMetric, "Memory consumption of the action containers", actionNamespace, initiatorNamespace, actionName)
```

Comment:
Updated the expression and also changed the chart title to `Memory` instead of `Max Memory`

![image](https://user-images.githubusercontent.com/664531/67176584-43fb0f80-f3e8-11e9-9236-2353cc32db1b.png)